frontend/src/packages/articles/articles-api.package.ts

```diff
@@ -34,6 +34,16 @@ class ArticleApi extends HttpApi {
 
     return await response.json<ArticleBaseResponseDto>();
   }
+
+  public async getArticle(id: string): Promise<ArticleBaseResponseDto> {
+    const articleApiPath = ArticlesApiPath.ROOT + id;
+    const response = await this.load(this.getFullEndpoint(articleApiPath, {}), {
```

Comment:
May replace :id from ApiPath with parameter you're passing
```suggestion
    const response = await this.load(this.getFullEndpoint(ArticlesApiPath.$ID, { id }), {
```